index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="stylesheet" href="css/font-awesome.min.css" />
+  </head>
+
+  <body>
+    <div class="nav-bar">
+      <h2>TEXT</h2>
+      <nav id="navbar">
+        <ul>
+          <li><a href="#welcome-section"> Home</a></li>
+          <li><a href="#projects"> Projects</a></li>
+          <li><a href="#contacts"> Contact me</a></li>
+        </ul>
+      </nav>
+    </div>
+
+    <div id="welcome-section">
+      <h1>TEXT</h1>
+      <p>Software developer</p>
+    </div>
+
+    <div id="projects">
+      <div class="projects-container">
+        <div class="project-tile">
+          <h3>Project 1</h3>
+          <a href="#">Link</a>
+        </div>
+        <div class="project-tile">
+          <h3>Project 2</h3>
+          <a href="#">Link</a>
+        </div>
+        <div class="project-tile">
+          <h3>Project 3</h3>
+          <a href="#">Link</a>
+        </div>
+        <div class="project-tile">
+          <h3>Project 4</h3>
+          <a href="#">Link</a>
+        </div>
+        <div class="project-tile">
+          <h3>Project 5</h3>
+          <a href="#">Link</a>
+        </div>
+        <div class="project-tile">
+          <h3>Project 6</h3>
+          <a href="#">Link</a>
+        </div>
+      </div>
+    </div>
+
+    <div class="contact" id="contacts">
+      <h3>Work with us!</h3>
+      <span id="profile-links">
+        <a id="profile-link" href="#" target="_blank">
+          <img src="./img/github.png" />
+        </a>
+        <a id="profile-link" href="#" target="_blank">
+          <img src="./img/linkedin.png" />
+        </a>
+        <a id="profile-link" href="#" target="_blank">
+          <img src="./img/facebook.png" />
+        </a>
+      </span>
```

Comment:
I think it's better to structure this piece of code as an unordered list `ul`, instead of using the `span` just for styling this section!

Reply:
Kindly provide a reason for the requested change.